modules/visual_script/config.py

```diff
@@ -43,6 +43,7 @@ def get_doc_classes():
         "VisualScriptReturn",
         "VisualScriptSceneNode",
         "VisualScriptSceneTree",
+        "VisualScriptScriptClass",
```

Comment:
Should this be `VisualScriptGlobalClass`?

Reply:
The terminology in the code base is kinda split between "script class", "global class", or, if a local variable/function is already using a similar name, explicitly "global script class". Generally speaking, I prefer using the "script class" terminology since it clearly identifies that the class in question is represented using a script. There's just an implication that script classes are globally named script resources.